gsy_framework/sim_results/all_results.py

```diff
@@ -28,7 +34,9 @@ def __init__(self, should_export_plots: bool = False, is_scm: bool = False):
             "cumulative_bills": CumulativeBills(),
             "cumulative_grid_trades": CumulativeGridTrades(),
             "device_statistics": DeviceStatistics(should_export_plots),
+            "forward_device_statistics": ForwardMarketDeviceStatistics(should_export_plots),
```

Comment:
Is it possible to make these new keys optional, to not populate these statistics for simulations without forward markets? So for example we could add these keys later, only when there is `ConstSettings.ForwardMarketSettings.ENABLE_FORWARD_MARKETS`.

Reply:
Well, yes. And to be honest, based on our last discussion with Spyros, I'm about to do so.